src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+
+import { footerData } from "../../constans.ts";
+import { themes } from "../../themes";
+import { Typography } from "../typography/Typography";
+import * as S from "./Footer.styled.ts";
+
+export const Footer = () => {
+	return (
+		<S.FooterWrapContainer>
+			<Typography
+				variant="h6"
```

Comment:
Brakuje tutaj "tag" albo "as" nie pamietam propsa, powinien to być znacznik HTML który odpowiada strukturze - kolejny poziom czyli h2, moze h3, moze h4?